src/development/platform-integration/ios/apple-frameworks.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Leveraging Apple's System Libraries
+description: Learn how to apply knowledge of Apple's APIs and frameworks when building Flutter apps.
+---
+
+This document is for iOS developers looking to apply
+their existing knowledge of Apple's APIs within Flutter.
+This includes accessing the device hardware and interacting 
+with specific frameworks like `HealthKit` or `MapKit`.
+
+<!-- Add once SwiftUI PR is merged -->
+<!-- For an overview of how the SwiftUI framework compares to Flutter, 
+see [Learning Flutter as a SwiftUI developer][].  -->
+
+### Flutter plugin overview
+When developing an app with Flutter, you can interact 
+with system libraries using plugins. 
+Plugins are Dart libraries that contain platform-specific code. 
+In the case of an iOS app, 
+the platform code can be written in Objective-C or Swift. 
+
+In your Dart code, you use the plugin's Dart API and 
+behind the scenes the native plugin code will directly call 
+the system library being used. This means that you can write 
+the code to call the Dart API once, 
+and it will work for all the platforms 
+that the plugin has been written for. 
+
+To learn more about plugins see [Using packages][]. 
+This document links out to some commonly used plugins, 
+and their examples, but there are thousands more available on [pub.dev][]. 
+And, if you can't find what you're looking for, you can create your own. 
+
+### Adding a plugin to your project
+When using Apple frameworks within your native project, 
+you usually just need to import it into your Swift or Objective-C file. 
+
+In Flutter, you must first run `flutter pub add package_name` 
+in the terminal, from the root of your project. This adds 
+the dependency to your [`pubspec.yaml`][] file. You can then import 
+the package in your Dart file. 
+
+Occasionally, you may also need to change app settings or 
```

Comment:
**question:** Can you be more specific here? Are there explicit cases when the dev would need to change the settings?

Reply:
It's dependent on the plugin. See an example here - https://pub.dev/packages/image_picker#ios

The dev needs to change settings by adding keys to info.plist